website/docs/cli-reference.md

```diff
@@ -276,6 +276,59 @@ Custom signer class module path.
 Waits for transaction to be accepted on chain.
 ### `migrate-configuration-file`
 Migrate protostar.toml V1 to V2.
+### `multicall`
+Execute multiple deploy (via UDC) and invoke calls ensuring atomicity.
+#### `file PATH`
+Required.
+
+Path to a TOML file with call declarations. File example:
+
+```toml
```

Comment:
https://github.com/software-mansion/protostar/discussions/1123#discussioncomment-4107492

Reply:
> YAML is used here instead of JSON to make appending easier

I think we should support JSON first and eventually add  TOML support (and add it everywhere). User writing a script using the cli will need two separate modules to parse two different formats. 

I agree with TOML benefits, but i believe consistency is important here.

@Arcticae WDYT?

Reply:
cc @mkaput

Reply:
@mkaput is off unfortunately :( @Arcticae what is your opinion here

Reply:
> I think we should support JSON first and eventually add TOML support (and add it everywhere). User writing a script using the cli will need two separate modules to parse two different formats.

A user who writes scripts doesn't need a separate module unless they want to read the file. They can create a template and replace certain values in it.

> I agree with TOML benefits, but i believe consistency is important here.

TOML is more consistent for users who don't write scripts, because of the configuration file.

Reply:
Can we just add support for JSON before we release the feature? It can be done in a separate task.

Reply:
>A user who writes scripts doesn't need a separate module unless they want to read the file. They can create a template and replace certain values in it.

This it doesn't work well with our general easy-to-script philosophy

Reply:
I created a task for it.
https://github.com/software-mansion/protostar/issues/1241

IMO TOML doesn't make it harder to script because it's an input file, not an output.